Doc/whatsnew/3.15.rst

```diff
@@ -263,6 +263,14 @@ http.client
   (Contributed by Alexander Enrique Urieles Nieto in :gh:`131724`.)
 
 
+http.cookies
+------------
+
+* Allow ``"`` double quotes in cookie values with a more linient regex for
+  double-quoted strings.
```

Comment:
Can we be more specific about what's changed than "a more lenient regex"? For example, do we now properly conform to an RFC/web specification? Are there restrictions that still exist on double-quoted strings?
```suggestion
* Allow '``"``' double quotes in cookie values with a more lenient regex for
  double-quoted strings.
```